ChangeLog.d/tls13-misc.txt

```diff
@@ -0,0 +1,9 @@
+Features
+   * Mbed TLS supports TLS 1.3 key establishment via pre-shared keys,
+     pre-shared keys provisioned externally or via the ticket mechanism
+     (session resumption).
+     The MBEDTLS_SSL_SESSION_TICKETS configuration option controls the support
+     of the ticket mechanism.
+     MBEDTLS_SSL_TLS1_3_KEY_EXCHANGE_MODE_xxx_ENABLED configuration options
+     have been introduced to control the support of the three possible
```

Comment:
```suggestion
     have been introduced to control the support for the three possible
```

Reply:
Indeed, it seems I have difficulties to assimilate this.